lib/msal-browser/src/config/Configuration.ts

```diff
@@ -240,6 +245,7 @@ export function buildConfiguration(
         navigateToLoginRequestUrl: true,
         clientCapabilities: [],
         protocolMode: ProtocolMode.AAD,
+        OIDCOptions: null,
```

Comment:
Please give this a default value of type `OIDCOptions`

Reply:
The reason I have it like this is because, as I understand, we don't want to set OIDCOptions to anything when using AAD protocol mode (in fact, we should be throwing an error if this happens). This is also why I have the null type, because it throws an error when I try to set it as just undefined.

Reply:
You can do both - throw an error here if AAD protocol mode is provided with OIDCOptions. And if it's not provided (regardless) give it a default value that matches our default behavior today (ResponseMode Fragment and existing default scopes)